test/integrations/react-test.js

```diff
@@ -252,7 +351,8 @@ describe('with multiple targets', () => {
 
 it('resolves assets correctly', async () => {
   await subject();
-  const zip = new AdmZip(config.targets.chrome.assetsPackage);
```

Comment:
This was pretty confusing for me, but I believe that the previous version of the tests were cramming in the location of the local zip file in the API response, which the domRunner would then eventually store in the chrome target under the assetsPackage property.

In this commit I ended up changing it so the mock server responds in the same way that the actual server would, which means that the path in the response is now where it is stored on S3. And then to find the zip file, I'm looking in the formData that was posted in the request more directly.

This makes these tests more realistic and removes a number of layers of confusing indirection.